charts/datadog/templates/cluster-agent-config-configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.clusterAgent.datadog_cluster_yaml }}
+{{- if or (.Values.clusterAgent.datadog_cluster_yaml) (eq (include "should-add-additional-prometheus-configs" .) "true") }}
```

Comment:
As it's repeated many times, I think the helper should rather be "should-use-configmap" or something

Reply:
yes good point thanks, we'll need two separate helpers though (for the DCA and the Agent)

Reply:
done